destinationCreation.html

```diff
@@ -0,0 +1,55 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+      <meta charset="UTF-8"/>
+      <title>Destination Creation</title>
+      <link rel="stylesheet" href="destinationCreation.css"/>
+      <script src="destinationCreation.js"></script>
+    </head>
+    <header>
+    </header>
+    <body>
+      <div id="content">
+        <h1>Destination Creation Form</h1>
+        <div id="form-container">
+          <form action="/data" method="post">
+            <label for="name-input">Enter the name of the destination: *</label>
```

Comment:
Instead of adding a "*" to each required input, you can do this with a class like this:

```
.required {
  color: red;
}
.required::after {
  content: "*";
}
```
You can make a required class and add it to the labels that are required.  This will let you style all the required labels the same way and add the little asterisk at the end!
https://www.w3schools.com/cssref/sel_after.asp